proposal/string-split.md

```diff
@@ -0,0 +1,105 @@
+# Add split function to Strings Module: Draft 1
+
+*([Issue](https://github.com/sass/sass/issues/1950))*
+
+This proposal adds `string.split()` to the `sass:string` module.
+
+## Table of Contents
+
+* [Background](#background)
+* [Summary](#summary)
+* [Semantics](#semantics)
+  * [`split()`](#split)
+
+## Background
+
+> This section is non-normative.
+
+The `sass:string` module contains several functions for
```

Comment:
Nit: Can you reflow this to 80-character lines? These first few paragraphs use shorter lines than necessary and a few of the lines down below are longer than 80 characters.

Reply:
Reflowed. All lines should be the correct length now.